hathor/indexes/memory_height_index.py

```diff
@@ -68,6 +68,19 @@ def get(self, height: int) -> Optional[bytes]:
     def get_tip(self) -> bytes:
         return self._index[-1].hash
 
-    def get_height_tip(self) -> tuple[int, bytes]:
+    def get_height_tip(self) -> HeightInfo:
         height = len(self._index) - 1
-        return height, self._index[height].hash
+        return HeightInfo(height, self._index[height].hash)
+
+    def get_n_height_tips(self, n_blocks: int) -> list[HeightInfo]:
+        if n_blocks < 1:
+            raise ValueError('n_blocks must be a positive, non-zero, integer')
+        # highest height that is included, will be the first element
+        h_high = len(self._index) - 1
+        # lowest height that is not included, None if it reaches the genesis
+        _h_low = h_high - n_blocks
+        h_low: Optional[int] = None if _h_low < 0 else _h_low
+        # zip the height using a range that starts at h_high and goes down to 0 and a slice of the index with len of at
+        # most n_blocks, both the range and the slice are already on the order that will be returned
+        return [HeightInfo(h, i.hash) for h, i in zip(range(h_high, -1, -1),
```

Comment:
Why not `[HeightInfo(h, self._index[h]) for h in range(h_high, h_low, -1)]`?

Reply:
Done.

Reply:
Thanks!